src/services/ui/src/features/package/package-activity/index.tsx

```diff
@@ -25,7 +25,24 @@ export const PA_RemoveAppkChild: FC<opensearch.changelog.Document> = (
       >
         {props.appkChildId}
       </Link>
-      <p>was withdrawn</p>
+      <p>was removed</p>
+    </div>
+  );
+};
+
+export const PA_AppkChildRemovedFromParent: FC<
+  opensearch.changelog.Document
+> = (props) => {
+  return (
+    <div className="flex gap-1">
+      <p>Removed from:</p>
+      <Link
+        path="/details"
+        query={{ id: props.appkParentId }}
+        className="hover:underline font-semibold text-blue-600"
+      >
+        {props.appkParentId}
+      </Link>
```

Comment:
Added new appk changelog reference for Withdrawn child.  Points to original parent